### PR TITLE
made changes from master to controlplane

### PR DESCRIPTION
**Pull request by gooney152**
#### What type of PR is this?
/kind cleanup
/wg naming
/sig testing
/area test

#### What this PR does / why we need it:
Refactor variables names that contains "master' to "controlplane".

#### Which issue(s) this PR fixes:
None

Fixes #

#### Does this PR introduce a user-facing change?
NONE

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by gooney152** [comment]
@oomichi [https://github.com/kubernetes/kubernetes/pull/103236] renaming

**Comment by gooney152** [comment]
@SataQiu  changed master variables names https://github.com/kubernetes/kubernetes/pull/103236

File: test/e2e/framework/skipper/skipper.go [hunk 1]
```diff
@@ -224,10 +224,10 @@ func SkipIfMultizone(c clientset.Interface) {
...
 }
 
-// SkipUnlessMasterOSDistroIs skips if the master OS distro is not included in the supportedMasterOsDistros.
-func SkipUnlessMasterOSDistroIs(supportedMasterOsDistros ...string) {
-	if !framework.MasterOSDistroIs(supportedMasterOsDistros...) {
-		skipInternalf(1, "Only supported for master OS distro %v (not %s)", supportedMasterOsDistros, framework.TestContext.MasterOSDistro)
+// SkipUnlessControlPlaneOSDistroIs skips if the controlplane OS distro is not included in the supportedControlPlaneOsDistros.
+func SkipUnlessControlPlaneOSDistroIs(supportedControlPlaneOsDistros ...string) {
+	if !framework.ControlPlaneOSDistroIs(supportedControlPlaneOsDistros...) {
+		skipInternalf(1, "Only supported for controlplane OS distro %v (not %s)", supportedControlPlaneOsDistros, framework.TestContext.ControlPlaneOSDistro)
```
Comment (wangyysde) [inline test/e2e/framework/skipper/skipper.go:230]: I found that you have renamed SkipUnlessMasterOSDistroIs to SkipUnlessControlPlaneOSDistroIs. But I can not find where is calling SkipUnlessControlPlaneOSDistroIs.

File: test/e2e/framework/test_context.go [hunk 2]
```diff
@@ -243,17 +243,17 @@ type CloudConfig struct {
 	Zones             []string // for multizone tests, use this set of zones instead of querying the cloud provider. Must include Zone.
 	Region            string
 	MultiZone         bool
-	MultiMaster       bool
+	MultiControlPlane       bool
 	Cluster           string
```
Comment (wangyysde) [inline test/e2e/framework/test_context.go:247]: nil: double space

File: test/e2e/framework/test_context.go [hunk 3]
```diff
@@ -243,17 +243,17 @@ type CloudConfig struct {
...
+	ControlPlaneName        string
 	NodeInstanceGroup string // comma-delimited list of groups' names
 	NumNodes          int
 	ClusterIPRange    string
 	ClusterTag        string
 	Network           string
 	ConfigFile        string // for azure and openstack
 	NodeTag           string
-	MasterTag         string
+	ControlPlaneTag         string
```
Comment (wangyysde) [inline test/e2e/framework/test_context.go:256]: as above

File: test/e2e/framework/test_context.go [hunk 4]
```diff
@@ -243,17 +243,17 @@ type CloudConfig struct {
 	Zones             []string // for multizone tests, use this set of zones instead of querying the cloud provider. Must include Zone.
 	Region            string
 	MultiZone         bool
-	MultiMaster       bool
+	MultiControlPlane       bool
 	Cluster           string
-	MasterName        string
+	ControlPlaneName        string
 	NodeInstanceGroup string // comma-delimited list of groups' names
```
Comment (wangyysde) [inline test/e2e/framework/test_context.go:249]: as above

**Pull request closed without merging**